### PR TITLE
[PORT] Medipen and regen cores can now be used in hand

### PR DESCRIPTION
**Pull request by Moccha-Bee**
## About The Pull Request

Makes you able to use medipens (all kinds) and regen cores in hand to apply them to yourself. Having gone mining a fair bit lately, I have come across some issues with being able to see/hit your sprite when north of megafauna due to their enormous sprites. Dying due to slowdown but with an unused regen core in hand doesn't feel good.
I recognize this is a slight buff to some of these items, but I think it's a necessary one.

Also removed some cyborg stuff from the medipens, these aren't used anymore to my knowledge.

Tried to do the autodoc thing, let me know if that's not how you want it.
## Why It's Good For The Game
Slight unclunk.

## Changelog
:cl:
balance: Medipens and legion cores can now be used in hand. This applies them to yourself, just like clicking your sprite.
removed: species mutator medipen.
/:cl:

### CODE REVIEW

File: code/modules/reagents/reagent_containers/hypospray.dm [hunk 1]
```diff
@@ -35,20 +39,20 @@
 	if(reagents.total_volume && (ignore_flags || M.can_inject(user, 1))) // Ignore flag should be checked first or there will be an error message.
 		to_chat(M, "<span class='warning'>You feel a tiny prick!</span>")
 		to_chat(user, "<span class='notice'>You inject [M] with [src].</span>")
-
```
Comment (eLeCtrOssSnake) [inline code/modules/reagents/reagent_containers/hypospray.dm:38]: Why did you delete this whitespace?

File: code/modules/reagents/reagent_containers/hypospray.dm [hunk 2]
```diff
@@ -35,20 +39,20 @@
...
 		var/fraction = min(amount_per_transfer_from_this/reagents.total_volume, 1)
 		reagents.reaction(M, INJECT, fraction)
+		
 		if(M.reagents)
 			var/trans = 0
 			if(!infinite)
 				trans = reagents.trans_to(M, amount_per_transfer_from_this, transfered_by = user)
 			else
 				trans = reagents.copy_to(M, amount_per_transfer_from_this)
-
```
Comment (eLeCtrOssSnake) [inline code/modules/reagents/reagent_containers/hypospray.dm:47]: Why did you delete this whitespace?

File: code/modules/reagents/reagent_containers/hypospray.dm [hunk 3]
```diff
@@ -201,13 +198,12 @@
...
-		literally appropriating other races."
-	volume = 1
-	amount_per_transfer_from_this = 1
-	list_reagents = list("unstablemutationtoxin" = 1)
+/obj/item/reagent_containers/hypospray/combat/heresypurge
+	name = "holy water autoinjector"
+	desc = "A modified air-needle autoinjector for use in combat situations. Prefilled with 5 doses of a holy water mixture."
+	volume = 250
+	list_reagents = list(/datum/reagent/water/holywater = 150, /datum/reagent/peaceborg/tire = 50, /datum/reagent/peaceborg/confuse = 50)
+	amount_per_transfer_from_this = 50
```
Comment (eLeCtrOssSnake) [inline code/modules/reagents/reagent_containers/hypospray.dm:206]: It takes 50u from a beaker using all it's mixture, so it will have a weird float with "/datum/reagent/peaceborg/tire", "/datum/reagent/peaceborg/confuse". Although it might not be subject to change.

File: code/modules/reagents/reagent_containers/hypospray.dm [hunk 4]
```diff
@@ -201,13 +198,12 @@
 	amount_per_transfer_from_this = 57
 	list_reagents = list(/datum/reagent/medicine/salbutamol = 10, /datum/reagent/medicine/leporazine = 15, /datum/reagent/medicine/tricordrazine = 15, /datum/reagent/medicine/epinephrine = 10, /datum/reagent/medicine/lavaland_extract = 2, /datum/reagent/medicine/omnizine = 5)
 
-/obj/item/reagent_containers/hypospray/medipen/species_mutator
-	name = "species mutator medipen"
-	desc = "Embark on a whirlwind tour of racial insensitivity by \
-		literally appropriating other races."
-	volume = 1
-	amount_per_transfer_from_this = 1
-	list_reagents = list("unstablemutationtoxin" = 1)
```
Comment (eLeCtrOssSnake) [inline code/modules/reagents/reagent_containers/hypospray.dm:210]: You deleted species mutator without commenting this in CL.

**Review by eLeCtrOssSnake — CHANGES_REQUESTED** [review]
You mostly have whitespace issues, consider looking in-depth what are you deleting.
Might need to note some pens deletions in CL.

File: code/modules/reagents/reagent_containers/hypospray.dm (hunk 4 above)
Comment (Moccha-Bee) [reply code/modules/reagents/reagent_containers/hypospray.dm:210]: Sure

**Comment by MarkSuckerberg** [comment]
Can I ask why exactly the species mutator pen was removed?

**Comment by Moccha-Bee** [comment]
I wont be able to do any chanes to this pr and it will become stale so i would rather close it.
